ipyvolume/pylab.py

```diff
@@ -389,20 +392,26 @@ def scatter(x, y, z, color=default_color, size=default_size, size_selected=defau
     :param size: {size}
     :param size_selected: like size, but for selected glyphs
     :param color_selected:  like color, but for selected glyphs
-    :param marker: {marker}
+    :param marker: Uses marker
     :param selection: numpy array of shape (N,) or (S, N) with indices of x,y,z arrays of the selected markers, which can have a different size and color
     :param kwargs:
     :return: :any:`Scatter`
     """
     fig = gcf()
     _grow_limits(x, y, z)
-    scatter = ipv.Scatter(x=x, y=y, z=z, color=color, size=size, color_selected=color_selected,
-                             size_selected=size_selected, geo=marker, selection=selection, **kwargs)
-    fig.scatters = fig.scatters + [scatter]
-    return scatter
+    if 'geo' in kwargs:
```

Comment:
'geo' is used for the Scatter object, but I think marker is a better name, and for this API i chose marker, which is taking it's role.

Reply:
So I'd remove this part, and simply use marker.

Reply:
since under the current formulation, `kwargs` go on to the scatter call, if someone does use `geo` it will result in a clash, since `marker` gets converted to `geo`. this simply says if some puts in `geo` it will overwrite `marker`

Reply:
Lets us remove this, and change to use geo->marker (or leave that for me to do)